function/index.js

```diff
@@ -44,6 +44,27 @@ function handleTempusBrokerMessage(data) {
     return s3Keys;
 }
 
+
+function getAnswerFromThemes (themes, themeId, answerId) {
+    return themes.find(theme => theme.id === themeId)
+        ?.values?.find(answer => answer.id === answerId)
+        ?.value;
+}
+
+function getApplicationOrigin(applicationData){
+    const testEmails = process.env.TEST_EMAILS || '';
+
+    const themesArray = applicationData.themes || [];
```

Comment:
I _think_ if `themes` isn't available then there is a bigger problem. If that's the case, it's probably best to let it error and the try block can pick it up